ckanext/canada/controller.py

```diff
@@ -458,11 +458,18 @@ def output_feed(self, results, feed_title, feed_description,
             last_page=navigation_urls['last'],
         )
 
+        chromos = h.recombinant_get_types()
+
         for pkg in results:
+            title, notes = None, None
+            if pkg['type'] in chromos:
+                chromo = h.recombinant_get_chromo(pkg['type'])
+                title= _(chromo['title'])
+                notes = _(pkg['notes'])
```

Comment:
they get updated by running `paster recombinant create <type>` after the description changes. Hacking it into the templates is something I'd like to avoid